### DISCUSSION TITLE
AJV Options

### DISCUSSION THREAD

**Discussion by spartyann** [josdejong/svelte-jsoneditor#129] [Ideas]
Hello,
(I am French, sorry for my bad English 😅)
I need to provide some specific parameters to AJV.
Would it be possible to add in `src/lib/plugins/validator/createAjvValidator.ts` a third parameter `ajvOptions` to the  constructor ?
This is a need in a professional context.
Example:

```
export function createAjvValidator(
  schema: JSONData,
  schemaDefinitions: JSONData = undefined,
  ajvOptions = {}  // <--------
): Validator {

  const ajv = new Ajv({
    allErrors: true,
    verbose: true,
    $data: true,
    ...ajvOptions  // <--------
  })

```

Thank you very much !!
Yann

**Comment by josdejong** [comment]
That makes sense, thanks for your suggestion 👍

**Reply by josdejong** [reply]
Implemented via https://github.com/josdejong/svelte-jsoneditor/commit/a66f230998a4e8c52a65d7cc5ce124968dec600f (not yet published)

**Reply by spartyann** [reply]
This is great! Thanks a lot!

**Reply by josdejong** [reply]
Published now in `v0.6.5`
